### DISCUSSION TITLE
How do i get BDFR to output links instead of downloading?

### DISCUSSION THREAD

**Discussion by TechWithTy** [Serene-Arc/bulk-downloader-for-reddit#832] [Q&A]
Looking to access **links to assets programmatically** without downloading for obvious reasons. 

`os send cmd bdfr -llinksonly earthporn  {params}`

i would like this to out put links to the media for the params i imput

**Comment by Serene-Arc** [comment]
This counts as metadata for the post, which is serviced by the `archive` command. It will download all of the metadata about the post and save it in a variety of data formats.

**Comment by Fakeaccount12312** [comment]
You could use the same method as in #835, but instead filter the log for the resource URLs, for example with this command:
```
grep "with url" log.txt | sed 's/.* //g' >> posts.txt
```
This should do the job without having to download all the jsons and parsing them.
